src/include/ipc/probe.h

```diff
@@ -0,0 +1,78 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Adrian Bonislawski <adrian.bonislawski@linux.intel.com>
+ *         Artur Kloniecki <arturx.kloniecki@linux.intel.com>
+ */
+
+/**
+ * \file include/ipc/probe.h
+ * \brief Probe IPC definitions
+ * \author Adrian Bonislawski <adrian.bonislawski@linux.intel.com>
+ * \author Artur Kloniecki <arturx.kloniecki@linux.intel.com>
+ */
+
+#ifndef __IPC_PROBE_H__
+#define __IPC_PROBE_H__
+
+#include <ipc/header.h>
+#include <stdint.h>
+
+/* Header for data packets sent via compressed PCM from extraction probes */
+struct probe_data_packet {
```

Comment:
This looks like ABI ???

Reply:
@lgirdwood I would guess it is. This structure will serve as header in compressed extraction stream, for each data packet. Structure is very closely related to cAVS one.
Should we attach our ABI header on top of this structure?

Reply:
ABI now added to classifier. Yes, only if you need it compile this header.